### PR TITLE
Feat: No affiliation vouches

### PR DESCRIPTION
**Pull request by mateodaza**

### CODE REVIEW

File: src/components/Modal/AttestModal.tsx/AttestModal.tsx [hunk 1]
```diff
@@ -279,7 +293,10 @@ export const AttestModal: FC<AttestModalProps> = ({
 							className='flex-1'
 							onClick={handleConfirm}
 							loading={step === AttestSteps.ATTESTING}
-							disabled={!selectedOrg || isCommentExceed}
+							disabled={
+								!userNoAffiliated &&
+								(!selectedOrg || isCommentExceed)
```
Comment (aminlatifi) [inline src/components/Modal/AttestModal.tsx/AttestModal.tsx:298]: The `isCommentExceed` can happen when the user is on no affiliation

File: src/components/Modal/AttestModal.tsx/AttestModal.tsx [hunk 2]
```diff
@@ -220,8 +237,9 @@ export const AttestModal: FC<AttestModalProps> = ({
 						<div className='border p-4'>
 							{isLoading ? (
 								<div>Loading user&apos;s organizations</div>
-							) : data && data?.length > 0 ? (
-								data?.map(ao => (
+							) : fetchedOrganisations &&
```
Comment (aminlatifi) [inline src/components/Modal/AttestModal.tsx/AttestModal.tsx:240]: `fetchedOrganisations &&` is extra, if `fetchedOrganisations` is undefined, `fetchedOrganisations?.length > 0` will be false already

File: src/components/Modal/AttestModal.tsx/AttestModal.tsx [hunk 3]
```diff
@@ -75,15 +87,20 @@ export const AttestModal: FC<AttestModalProps> = ({
...
 
 	const handleConfirm = async () => {
-		if (!address || !signer || !selectedOrg) return;
-
+		let _selectedOrg = selectedOrg;
+		if (!address || !signer) return;
+		if (userNoAffiliated) {
+			_selectedOrg = NO_AFFILIATED_ORG;
+		}
+		if (!_selectedOrg) return;
```
Comment (aminlatifi) [inline src/components/Modal/AttestModal.tsx/AttestModal.tsx:103]:
```suggestion
		if (!address || !signer) return;
		const _selectedOrg = userNoAffiliated ? NO_AFFILIATED_ORG : selectedOrg;
		if (!_selectedOrg) return;
```

**Review by aminlatifi — CHANGES_REQUESTED** [review]
Thanks @mateodaza 
Just few minor change requests

File: src/components/Modal/AttestModal.tsx/AttestModal.tsx (hunk 2 above)
Comment (mateodaza) [reply src/components/Modal/AttestModal.tsx/AttestModal.tsx:240]: eslint doesn't like that 🤣

**Pull request merged**